src/conn/conn_sweep.c

```diff
@@ -146,10 +161,17 @@ __sweep_expire(WT_SESSION_IMPL *session, uint64_t now)
               session, WT_WITH_DHANDLE(session, dhandle, ret = __sweep_expire_one(session)));
         else
             WT_WITH_DHANDLE(session, dhandle, ret = __sweep_expire_one(session));
-        WT_RET_BUSY_OK(ret);
+        WT_ERR_ERROR_OK(ret, EBUSY, true);
+    }
+err:
+    ret = 0;
```

Comment:
I don't really like setting `ret` to `0` here. I would do it after line 164. Or should you simply set `keep` to `false` instead?
```
        WT_ERR_ERROR_OK(ret, EBUSY, false);
```

Reply:
Done. You are right.